examples/multi-threaded/main.c

```diff
@@ -61,14 +63,18 @@ static void pcb(struct mg_connection *c, int ev, void *ev_data, void *fn_data) {
 static void fn(struct mg_connection *c, int ev, void *ev_data, void *fn_data) {
   if (ev == MG_EV_HTTP_MSG) {
     struct mg_http_message *hm = (struct mg_http_message *) ev_data;
+      char param[64];
+      mg_http_get_var(&hm->query, "param", param, sizeof(param));
+      if (param[0] == '\0') strcpy(param,"hi");
     if (mg_http_match_uri(hm, "/fast")) {
       // Single-threaded code path, for performance comparison
       // The /fast URI responds immediately
-      mg_http_reply(c, 200, "Host: foo.com\r\n", "hi\n");
+      mg_http_reply(c, 200, "Host: foo.com\r\n", param);
```

Comment:
I'd keep the response as "hi\n"

Reply:
OK, just thought both "fast" and "threaded" should do the same.